sci-biology/bowtie/files/bowtie-2.2.4-buildsystem.patch

```diff
@@ -1,19 +1,22 @@
---- Makefile.old	2013-09-22 12:07:12.606844000 +0200
-+++ Makefile	2013-09-22 13:06:03.113888200 +0200
-@@ -24,8 +24,6 @@
+--- Makefile
++++ Makefile
+@@ -22,11 +22,6 @@
+ #
+ 
  INC =
- GCC_PREFIX = $(shell dirname `which gcc`)
- GCC_SUFFIX =
+-GCC_PREFIX = $(shell dirname `which gcc`)
+-GCC_SUFFIX =
 -CC = $(GCC_PREFIX)/gcc$(GCC_SUFFIX)
 -CPP = $(GCC_PREFIX)/g++$(GCC_SUFFIX)
- CXX = $(CPP)
+-CXX = $(CPP)
  HEADERS = $(wildcard *.h)
  BOWTIE_MM = 1
-@@ -141,7 +139,7 @@
+ BOWTIE_SHARED_MEM = 0
+@@ -152,7 +147,7 @@
  
- DEBUG_FLAGS    = -O0 -g3 $(BITS_FLAG) $(SSE_FLAG)
+ DEBUG_FLAGS    = -O0 -g3 -m64 $(SSE_FLAG)
  DEBUG_DEFS     = -DCOMPILER_OPTIONS="\"$(DEBUG_FLAGS) $(EXTRA_FLAGS)\""
--RELEASE_FLAGS  = -O3 $(BITS_FLAG) $(SSE_FLAG) -funroll-loops -g3
+-RELEASE_FLAGS  = -O3 -m64 $(SSE_FLAG) -funroll-loops -g3
 +RELEASE_FLAGS  = $(CXXFLAGS)
  RELEASE_DEFS   = -DCOMPILER_OPTIONS="\"$(RELEASE_FLAGS) $(EXTRA_FLAGS)\""
  NOASSERT_FLAGS = -DNDEBUG
```

Comment:
Same here, all these flags could be overwritten from the ebuild and hence no patch would be necessary.